### PR TITLE
B 19827 int

### PR DESCRIPTION
**Pull request by deandreJones**
## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the admin app-
2. go to office users
3. see task invoicing officer instead of transportation invoicing officer
4. create a TIO - see that you are able to login successfully

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

## Screenshots

image
image
image

### CODE REVIEW

**Review by TevinAdams — APPROVED** [review]
Rename looks good. Went through admin and office and did not see any reference to Transportation Invoicing Officer. Was able to create a Task Invoicing Officer, Request the Task Invoicing role, and accept Office request with Task invoicing role. Was able to log in as a TIO and perform functions as usual.

**Review by cameroncaci — CHANGES_REQUESTED** [review]
"Transportation invoicing" is still present on the following lines:
`src/pages/Admin/RequestedOfficeUsers/RequestedOfficeUserEdit.jsx:51`
`src/pages/Admin/OfficeUsers/OfficeUserEdit.jsx:50`
`src/pages/Admin/OfficeUsers/OfficeUserCreate.jsx:33`

**Comment by deandreJones** [comment]
> "Transportation invoicing" is still present on the following lines: `src/pages/Admin/RequestedOfficeUsers/RequestedOfficeUserEdit.jsx:51` `src/pages/Admin/OfficeUsers/OfficeUserEdit.jsx:50` `src/pages/Admin/OfficeUsers/OfficeUserCreate.jsx:33`

hmm.. must hav been from a new Pr

**Comment by deandreJones** [comment]
> > "Transportation invoicing" is still present on the following lines: `src/pages/Admin/RequestedOfficeUsers/RequestedOfficeUserEdit.jsx:51` `src/pages/Admin/OfficeUsers/OfficeUserEdit.jsx:50` `src/pages/Admin/OfficeUsers/OfficeUserCreate.jsx:33`
> 
> hmm.. must hav been from a new Pr

ok - should be resolved now
@cameroncaci

File: .circleci/config.yml
Comment (traskowskycaci) [inline .circleci/config.yml:1]: Do we want/need to update the cached yarn.lock file?

**Review by traskowskycaci — APPROVED** [review]

**Review by cameroncaci — APPROVED** [review]
Looks good, AC is met!

File: .circleci/config.yml
Comment (deandreJones) [reply .circleci/config.yml:1]: actually no- I was trying something when the circle build was failing something that didnt exist on my local- turned out it was integration and not main.. 
but upping the version doesn't hurt anything- but gets us a fresh cache file-- with the issues we ran into last week - not a bad idea to get fresh cache

**Pull request merged**